frontend/src/pages/hardwareProfiles/HardwareProfilesTableRow.tsx

```diff
@@ -38,6 +38,7 @@ type HardwareProfilesTableRowProps = {
   rowIndex: number;
   hardwareProfile: HardwareProfileKind;
   migrationAction?: MigrationAction;
+  isMigratedTable?: boolean;
```

Comment:
It doesn't need to be an optional prop here. It has a default value in `HardwareProfilesTable` already, so this will always be provided. Then you don't need to give it a default value below.